contracts/ExampleToken.cdc

```diff
@@ -44,228 +78,148 @@ pub contract ExampleToken: FungibleToken {
     /// out of thin air. A special Minter resource needs to be defined to mint
     /// new tokens.
     ///
-    pub resource Vault: FungibleToken.Provider, FungibleToken.Receiver, FungibleToken.Balance, MetadataViews.Resolver {
+    access(all) resource Vault: FungibleToken.Vault {
 
         /// The total balance of this vault
-        pub var balance: UFix64
+        access(all) var balance: UFix64
+
+        access(all) var storagePath: StoragePath
+        access(all) var publicPath: PublicPath
+        access(all) var receiverPath: PublicPath
```

Comment:
yeah, when I originally made this, I wrote it assuming that this would just be an example for new developers to follow writing new contracts. I wasn't thinking about migrations since these are just example contracts. But I understand that some developers will copy this when working on their migrations, so do you think it is worth changing all of this so it is a better example for migrations? We ended up doing that for `ExampleNFT`, but I don't know how important it is for `ExampleToken` since there are so few fungible tokens in the ecosystem right now

Reply:
Makes sense. I think the simplest approach is to leave the paths as contract fields unless there's a strong reason to make them Collection fields, but I don't think there's harm done either way. Just called it out since it came up when migrating ExampleNFT.